README.md

```diff
@@ -1,13 +1,61 @@
 # ini parser
 
-## Try ini parser
+## How to use
 
-```bash
-go run main.go
+- Create a new *.ini file and insert your ini content, for example:
+  
+```ini
+[owner]
+name = John
+organization = threefold
+
+[database]
+version = 12.6
+server = 192.0.2.62
+port = 143
+password = 123456
+protected = true
+```
+
+- Create a new parser struct
+
+```go
+parser := ini.Parser{}
```

Comment:
Show how to import the package in github, and add `NewParser` function to your API